pyro/distributions/log_normal_negative_binomial.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+from numpy.polynomial.hermite import hermgauss
+
+import torch
+from torch.distributions import constraints
+from torch.distributions.utils import lazy_property, broadcast_all
+
+from pyro.distributions.torch_distribution import TorchDistribution
+from pyro.distributions.torch import NegativeBinomial
+from pyro.distributions.util import broadcast_shape
+
+
+def get_quad_rule(num_quad, prototype_tensor):
```

Comment:
should i put this in pyro ops or the like and add tests?

Reply:
It's up to you. If you foresee reusing it, feel free to move it. Otherwise I'd rename it to `_get_quad_rule()` to ensure it's private to this file.